<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jeu qui casse des briques</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>630</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>751</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_scene">
     <property name="spacing">
      <number>6</number>
     </property>
    </layout>
   </widget>
   <widget class="QComboBox" name="level_select">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>620</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Niveau 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Niveau 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Niveau 3</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="level_name">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>580</y>
      <width>121</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Choix du niveau :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restart_level">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>630</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>630</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="progress_name">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>171</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Progression du niveau :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
